### PR TITLE
feat: add config for Nuxt 3

### PR DESCRIPTION
**Pull request by GabriMcNab**
Added configuration for Nuxt 3. It's based on [@nuxt/eslint-config](https://github.com/nuxt/eslint-config#readme) which is the official plugin for the Nuxt community.

### CODE REVIEW

**Comment by aureliome** [comment]
Do we need to add `eslint-config-typescript` for Typescript projects, as shown here https://github.com/nuxt/eslint-config#typescript ?

File: lib/configs/nuxt3.js [hunk 1]
```diff
@@ -0,0 +1,30 @@
+const {
+  rootConfigFilesOverride,
+} = require("../shared/overrides/rootConfigFiles");
+
+module.exports = {
```
Comment (aureliome) [inline lib/configs/nuxt3.js:5]: Do we need to add eslint-config-typescript for Typescript projects, as shown here https://github.com/nuxt/eslint-config#typescript ?

File: lib/index.js [hunk 2]
```diff
@@ -5,6 +5,7 @@ module.exports = {
     library: require('./configs/library'),
     vue: require('./configs/vue'),
     nuxt: require('./configs/nuxt'),
+    nuxt3: require('./configs/nuxt3'),
```
Comment (aureliome) [inline lib/index.js:8]: Do we need to add a section to README.md with the instructions to use this eslint plugin with Nuxt3 projects?

**Review by aureliome — CHANGES_REQUESTED** [review]
Good job 😉

File: lib/configs/nuxt3.js (hunk 1 above)
Comment (GabriMcNab) [reply lib/configs/nuxt3.js:5]: The documentation is a little outdated...they currently export 3 packages:

Screenshot 2023-02-17 at 10 59 36

The first 2 are the ones for Nuxt2. The third one is the new config, and typescript is already included there: https://github.com/nuxt/eslint-config/blob/main/packages/eslint-config/index.js

File: lib/index.js (hunk 2 above)
Comment (GabriMcNab) [reply lib/index.js:8]: Yes, I will add it! Thanks!

**Review by aureliome — APPROVED** [review]

**Pull request merged**